cpp/src/io/comp/debrotli.cu

```diff
@@ -61,31 +61,65 @@ THE SOFTWARE.
 
 namespace cudf {
 namespace io {
-#define HUFFTAB_LUT1_BITS 8
-#define HUFFCODE(len, sym) ((uint16_t)(((sym) << 4) + (len)))
-#define BROTLI_CODE_LENGTH_CODES 18
-#define BROTLI_NUM_DISTANCE_SHORT_CODES 16
-#define BROTLI_MAX_ALLOWED_DISTANCE 0x7FFFFFFC
+constexpr uint32_t hufftab_lut1_bits               = 8;
+constexpr int8_t brotli_code_length_codes          = 18;
+constexpr uint32_t brotli_num_distance_short_codes = 16;
+constexpr uint32_t brotli_max_allowed_distance     = 0x7FFFFFFC;
+constexpr int numthreads                           = 256;
```

Comment:
I think most of the kernels use a name like `block_size` rather than `num_threads`. Let's be consistent. But at least separate words.

```suggestion
constexpr int num_threads                           = 256;
```